src/components/home/NewsItem.jsx

```diff
@@ -5,8 +5,7 @@ import Package from "../../../package.json"
 
 const NewsItem = () => (
   <div className="news-item" data-testid="news-item">
-    <h3>Latest news</h3>
```

Comment:
Should `<h3>Latest news</h3>` be deleted? It is causing a test failure on this PR.

Reply:
why is that a problem? is the test looking for that line? if so i'm fine to put it back